src/Microsoft.FSharp.Compiler/Microsoft.FSharp.Compiler.fsproj

```diff
@@ -65,7 +65,8 @@
   </ItemGroup>
 
   <Target Name="PackDependentProjectsCore">
-    <MSBuild Projects="@(DependentProjects)" Targets="Pack" Properties="Restore=true;Pack=true;BUILD_PUBLICSIGN=$(BUILD_PUBLICSIGN)" />
+    <!-- We embed nuget packages into the Microsoft.FSharp.Compiler nuget package. so when we pack the dependencies we need to do repack to make SHipping and Release versions -->
+    <MSBuild Projects="@(DependentProjects)" Targets="Pack" Properties="Restore=true;Pack=true;UsingToolNuGetRepack=true;BUILD_PUBLICSIGN=$(BUILD_PUBLICSIGN)" />
```

Comment:
I do not think you need the UsingToolNuGetRepack here.

Reply:
Oh I do so much more crazy now.